templates/python/.github/workflows/pre-commit.yaml

```
@@ -0,0 +1,49 @@
+name: Pre-commit
+
+on:
+  push:
+    # branches: [main]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```

tests should always run

```suggestion
    branches: [main]
    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```

---

Ah I have misunderstood how the tag branches works. I thought that if not specified the default value is the set of all branches.

---

uncommented